src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int NAME_POSITION = 0;
+    static final int NUM_POSITION = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        int sumSupply = 0;
+        int sumBuy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String value = reader.readLine();
+            while (value != null) {
```

Comment:
Consider trimming the `value` string to avoid issues with leading or trailing whitespace. This can prevent potential errors when splitting the string.